packaging/debian/changelog

```diff
@@ -1,3 +1,9 @@
+syslog-ng (3.35.1-1) unstable; urgency=medium
```

Comment:
I think we should change the `distribution` to `stable` as we use that name in the APT repo as well.